src/server/cache.rs

```diff
@@ -86,25 +88,35 @@ async fn tile_cache_updater(state: &ServerState) {
         match result {
             Ok(response) => {
                 //trace!("tile_cache_updater: {:#?}", response);
-                let tiles = match serde_json::to_string(&response) {
-                    Ok(tiles) => tiles,
+                let json_tiles = match serde_json::to_string(&response) {
```

Comment:
Until we know more about how large this cache will eventually be, in retrospect, storing solely the json in it is really a premature optimization on my part.

What do you think about just storing the `AdmTileResponse` struct and having it derive `Eq/PartialEq` (`AdmTile` needs them too). We could store the rendered json along side it too (maybe that's even premature before benchmarks)  -- but I'd rather we stick to comparing the cache values with strict equality (via `AdmTileResponse`) for now vs relying on a hash code (and potential collisions).

Maybe we'll need something like this later if the cache is too large but I'm leaning towards keeping this dead simple until we need something different.

Reply:
I'd be ok with using the `Eq/PartialEq` trait. I guess I was more thinking that we want to compare elements of the tile. (e.g. we may not need to compare `id` or `position` for instance.) This is kinda per the other comments. Honestly, i was more figuring that there could be variations in the returned tile values that might churn the cache and this way we'd have a normalization point.